src/main/java/gov/nasa/jpl/mbee/mdk/options/MDKOptionsGroup.java

```diff
@@ -138,6 +141,21 @@ public void setUserScriptDirectory(String path) {
         property.setGroup(GROUP);
         addProperty(property, true);
     }
+    
+    
+    public String getDefaultDocBookToPDFStyleSheet() {
```

Comment:
See proposed rename above.

Reply:
Remove the Default and change to getDocBookToPDFStyleSheet()